reader.go

```diff
@@ -90,9 +91,11 @@ func (rdr *LogLineReader) ReadLines() error {
 			rdr.linesRead.Inc(1)
 			rdr.mu.Lock()
 			if full := rdr.b.Add(LogLine{line, currentLogTime}); full {
+				rdr.batchFillTime.UpdateSince(now)
 				rdr.deliverOrDropCurrent()
 			}
 			if rdr.b.MsgCount() == 1 { // First line so restart the timer
+				now = time.Now()
```

Comment:
If the answer for my last comment is yes we could remove this assignment from here.